ArcGISRuntimeSDKQt_QMLSamples/Layers/RasterFunctionFile/RasterFunctionFile.qml

```diff
@@ -25,7 +25,7 @@ Rectangle {
     height: 600
 
     property real scaleFactor: 1
-    property url dataPath: System.userHomePath + "/ArcGIS/Runtime/Data/raster"
+    property url dataPath: System.writableLocationUrl(System.StandardPathsHomeLocation) + "/ArcGIS/Runtime/Data/raster"
```

Comment:
Ditto iOS conditional?

Since there are a few of these, I won't comment on them all.